### PR TITLE
cleanup introspector job log messages in modelInImage.sh

### PR DESCRIPTION
**Pull request by tbarnes-us**

### CODE REVIEW

File: operator/src/main/resources/scripts/modelInImage.sh [hunk 1]
```diff
@@ -227,9 +227,12 @@ function buildWDTParams_MD5() {
...
-    trace SEVERE "Domain Source Type is 'FromModel' and domain type JRF which requires specifying a " \
-       "walletPasswordSecret in your domain resource and deploying this secret with a 'walletPassword' key, " \
-       " but the secret does not have this key."
+    trace SEVERE "The domain resource 'spec.domainHomeSourceType'" \
+       "is 'FromModel' and the 'spec.configuration.model.domainType' is 'JRF';" \
+       "this combination requires specifying a" \
+       "'spec.configuration.model.walletPasswordSecret' in your domain" \
+       "resource and deploying this secret with a 'walletPassword' key," \
+       "but the secret does not have this key."
     exit 1
```
Comment (jshum2479) [inline operator/src/main/resources/scripts/modelInImage.sh:236]: it should be exitOrLoop for consistency

**Review by jshum2479 — COMMENTED** [review]
LGTM, I thought you are changing it also to use traceMII?

File: operator/src/main/resources/scripts/modelInImage.sh (hunk 1 above)
Comment (tbarnes-us) [reply operator/src/main/resources/scripts/modelInImage.sh]: done, plus i made the same change to another 'exit 1' in the same file

**Comment by tbarnes-us** [comment]
> LGTM, I thought you are changing it also to use traceMII?

I just updated utils.sh 'trace' with a universal fix so that it properly shows the script name of the caller,
so trace calls from modelInImage.sh now properly include a `[modelInImage.sh:NNN]`.
  
The old behavior displayed the script name of the caller incorrectly if the caller was 'sourced' in (it would
instead show the script name of the file that called source).

For example, here's the new behavior:

```
 ->  kubectl -n sample-domain1-ns logs -f sample-domain1-introspector-f2llp
@[2021-02-02T23:34:33.290 UTC][utils.sh:294][FINE] id = 'uid=1000(oracle) gid=0(root) groups=0(root)'
@[2021-02-02T23:34:33.714 UTC][introspectDomain.sh:110][FINE] Introspecting domain 'sample-domain1', log location: '/tmp/introspector_script.out'
...
@[2021-02-02T23:34:34.271 UTC][modelInImage.sh:102][FINE] Entering checkExistInventory
@[2021-02-02T23:34:34.288 UTC][modelInImage.sh:104][FINE] Checking wdt artifacts in image
...
@[2021-02-02T23:35:46.283 UTC][wlst.sh:54][FINE] WLST script '/weblogic-operator/scripts/introspectDomain.py' completed.
@[2021-02-02T23:35:46.303 UTC][introspectDomain.sh:233][FINE] Domain introspection complete

 ->  kubectl -n sample-domain1-ns logs -f sample-domain1-admin-server
script path is /weblogic-operator/scripts
@[2021-02-02T23:35:49.634 UTC][startServer.sh:21][FINE] Starting WebLogic Server 'admin-server'.
...
@[2021-02-02T23:35:49.773 UTC][startServer.sh:192][FINE] Model-in-Image: Restore domain secret
@[2021-02-02T23:35:49.791 UTC][modelInImage.sh:1089][FINE] Entering encrypt_decrypt_domain_secret decrypt
@[2021-02-02T23:35:52.345 UTC][modelInImage.sh:1127][FINE] Exiting encrypt_decrypt_domain_secret
@[2021-02-02T23:35:52.363 UTC][startServer.sh:205][FINE] Model-in-Image: Restore domain config
...
@[2021-02-02T23:35:52.813 UTC][startServer.sh:61][FINE] Start node manager
...
@[2021-02-02T23:35:52.836 UTC][startNodeManager.sh:68][FINE] Starting node manager for domain-uid='sample-domain1' and server='admin-server'.
...
@[2021-02-02T23:35:55.882 UTC][startServer.sh:98][FINE] Start WebLogic Server via the nodemanager
...
@[2021-02-02T23:35:55.981 UTC][wlst.sh:48][FINE] Running wlst script '/weblogic-operator/scripts/start-server.py'
...
<Feb 2, 2021 11:36:04 PM GMT> <INFO> <NodeManager> <Starting WebLogic server with command line: /u01/domains/sample-domain1/bin/startWebLogic.sh >
...
<Feb 2, 2021 11:36:07 PM GMT> <Info> <WebLogicServer> <BEA-000377> <Starting WebLogic Server with Java HotSpot(TM) 64-Bit Server VM Version 25.271-b09 from Oracle Corporation.>
...
<Feb 2, 2021 11:36:18,817 PM GMT> <Notice> <WebLogicServer> <BEA-000360> <The server started in RUNNING mode.>
...

```

**Comment by jshum2479** [comment]
Awesome! Thanks Tom.

**Pull request merged**